src/Plugin.NFC/iOS/NFC.iOS.cs

```diff
@@ -188,7 +204,13 @@ public override void DidDetectTags(NFCTagReaderSession session, INFCTag[] tags)
 						// if ndef is not supported do not read or write
 						// let the user decide if ndef support is needed
 						OnMessageReceived?.Invoke(nTag);
+
+						OnError?.Invoke(
```

Comment:
since we only read the identifier of the tag and not its content we could skip the check for proper NDEF format for now. I removed the invoke for onError. Done.